ui/appui-react/src/appui-react/icons/SvgSectionTool.tsx

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Utilities
+ */
+
+import * as React from "react";
+
+/** @internal */
+export function SvgSectionTool() {
+  return (
+    <svg
+      data-name="Calque 1"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 16 16"
+    >
```

Comment:
Was this SVG optimized?  I still see `data-name="Calque 1"` which should have been removed.

Reply:
The SVG was optimized both SVGOMG and SVGViewer didn't remove this property when optimizing. Removed the property now.